geomesa-archetypes/geomesa-archetypes-binary-dist/assembly.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/component/1.1.2"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/component/1.1.2 http://maven.apache.org/xsd/component-1.1.2.xsd">
```

Comment:
When you change the archetype, do you clean out and respin various part of the project to test it out?

I'm trying to understand the workflow with Maven archetypes.

Reply:
archetypes aren't really a maven thing, it's just the nomenclature i adopted. it's just used as the parent of all the -dist modules now, instead of e.g. geomesa-accumulo being the parent of geomesa-accumulo-dist, etc. i did test out all the binary dists and compared them to the previous way to ensure there weren't any unintentional differences

Reply:
i should say, maven archetypes are a thing, but not this thing ;)

maybe i should have named it something else...

Reply:
That clears it up!